tv-show/src/components/features/shows/ShowReviewSection/ShowReviewSection.tsx

```diff
@@ -12,12 +12,11 @@ interface IShowReviewSectionProps {
 
 export default function ShowReviewSection({reviews, onSubmit, user}: IShowReviewSectionProps) {
   return (
-    <Container>
-       <Heading size='md' marginBottom={4}>
-        Reviews
-      </Heading>
-      <Card variant='unstyled'>
-        <CardBody>
+    <Flex bg="darkPurple" >
```

Comment:
All the code I am looking at is good, but the styling is still messed up. This component is an example 
![image](https://github.com/user-attachments/assets/f12a0e73-f7e4-4f51-aed0-a9d497c7a37c)